sql/core/src/test/resources/sql-tests/results/literals.sql.out

```diff
@@ -197,9 +197,9 @@ select .e3
 -- !query 20
 select 1E309, -1E309
 -- !query 20 schema
-struct<Infinity:double,-Infinity:double>
+struct<1E+309:decimal(1,-309),-1E+309:decimal(1,-309)>
 -- !query 20 output
-Infinity	-Infinity
+1000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000	-1000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
```

Comment:
Ok, found that cause. `SQLQueryTestSuite` uses `QueryExecution.hiveResultString` to output its results; this writes a `BigDecimal` using `toPlainString` which in a completely written out value. I'd rather not change this since this is only used for Hive tests.